public/r2/v1/swagger/swagger.json

```diff
@@ -836,6 +836,9 @@
                 "name": {
                     "type": "string"
                 },
+                "description": {
```

Comment:
For this we actually need to make a change to the backend protobuf message.

Reply:
Yup, I wanted to make a separate pull request for that (as it's in a different repo I believe). This pull request was mainly to confirm  agree on the upcoming API change, and have the changes updated in Swagger.

Reply:
Cool, looks good. I agree with Xi, we should merge it with the actual implementation so its consistent on master.

Also, I wonder if there is a way to extend an Object in swagger markup. eg MappingRule is implemented by MappingRuleData. Might help on code duplication.